common/common-client/src/main/java/org/ow2/proactive/authentication/FileLoginModule.java

```diff
@@ -258,21 +256,17 @@ private boolean authenticateUserFromFile(String username, String password) throw
      * @throws LoginException if group file is not found or unreadable.
      */
     protected void groupMembershipFromFile(String username) throws LoginException {
-        Properties groups = new Properties();
-        try {
+
+        try (FileInputStream stream = new FileInputStream(groupFile)) {
+            BufferedReader reader = new BufferedReader(new InputStreamReader(stream, Charsets.UTF_8));
```

Comment:
It seems there exists a ProActive property that could, perhaps, be reused: `PAProperties#getFileEncoding`. However, after a quick look, I am not sure that setting file encoding is required. We define encoding at JVM startup to UTF-8, thus default encoding is UTF-8. Then, if charset is no set when creating an InputStreamReader, default encoding is used.